versions.yaml

```diff
@@ -33,7 +33,7 @@ docker_images:
   nginx:
     description: "Proxy server for HTTP, HTTPS, SMTP, POP3 and IMAP protocols"
     url: "https://hub.docker.com/_/nginx/"
-    version: "1.17.0-alpine"
+    version: "1.15-alpine"
```

Comment:
why we are using an older version of alpine?

Reply:
Because that's what I could find in quay.io. :-)
Is there the need to test specifically against 1.17.0-alpine of would this be okay?

Reply:
@fidencio not that I remembered, thanks